tools/vendor_hid_test.py

```diff
@@ -1,7 +1,12 @@
 """These tests verify the functionality of the VendorHID interface."""
-from typing import Dict, Iterable
```

Comment:
I will review this file after #520 landed, since the diff should be small? There is not commit with the exact diff.

Reply:
Yeah, sorry, I didn't base this PR off that one. 

The diff will be small once #520 lands.

Reply:
Done.